deploy/charts/cert-manager/values.yaml

```diff
@@ -236,14 +236,14 @@ enableCertificateOwnerRef: false
 #      StableCertificateRequestName: true
 #      UseCertificateRequestBasicConstraints: true
 #      ValidateCAA: true
+#    # Configure the metrics server for TLS
+#    # See https://cert-manager.io/docs/devops-tips/prometheus-metrics/#tls
 #    metricsTLSConfig:
 #      dynamic:
 #        secretNamespace: "cert-manager"
 #        secretName: "cert-manager-metrics-ca"
 #        dnsNames:
 #        - cert-manager-metrics
-#        - cert-manager-metrics.cert-manager
-#        - cert-manager-metrics.cert-manager.svc
```

Comment:
These extra DNS names do not seem to be necessary for either the PodMonitor or ServiceMonitor.
I verified that both work with only a single  `cert-manager-metrics` name.